ts/tile-cache.ts

```diff
@@ -287,9 +288,9 @@ export class IndexedDbTileCache extends EventEmitter {
      *
      * The returned number in the promise is equal to the duration of the operation in milliseconds.
      */
-    public seedBBox(bbox: IBBox, maxZ: number, minZ: number = 0): Promise<number> {
+    public seedBBox(bbox: IBBox, maxZ: number, minZ: number = 0, tms: boolean = false): Promise<number> {
         const start = Date.now();
-        const list: ITileCoordinates[] = getListOfTilesInBBox(bbox, maxZ, minZ);
+        const list: ITileCoordinates[] = getListOfTilesInBBox(bbox, maxZ, minZ, tms);
```

Comment:
You should also specify the new version of `@yaga/tile-utils`, that includes tms support in the `package.json`.

For that, you have to apply the changes according to my requests in [this PR](https://github.com/yagajs/tile-utils/pull/1/files). I will merge it as soon as possible and give you a new version number of `tile-utils` for this release. I think it will be `1.1.0`...